### ISSUE TITLE
Can you add this website?

### ISSUE THREAD

**Issue by plan291037** [ytdl-org/youtube-dl#30007]
www.bilibili.tv
I want to download subtitles from this website. Thanks

**Comment by PrivateGER** [comment]
This project is probably abandoned, check the yt-dlp fork, they have it implemented.

**Comment by dirkf** [comment]
The site was supported: see #29852. In the existing extractor, subtitles could be returned for an audio download with a `lyric`.

**Comment by plan291037** [comment]
> The site was supported: see #29852. In the existing extractor, subtitles could be returned for an audio download from bilibili.com with a `lyric`.
not .com it is .tv sir

**Comment by dirkf** [comment]
Indeed, subtitles don't seem to have been supported except in the case that I listed, although media from .tv were supported.

Probably easiest to pull the extractor from yt-dlp, now.

**Comment by Ashish0804** [comment]
> Probably easiest to pull the extractor from yt-dlp, now.

Hi
I wrote the PR supporting this site https://github.com/yt-dlp/yt-dlp/pull/907
While the code is released under unlicense and can be merged with yt-dl without any license issues, I'd appreciate if my code is not merged here.
Thanks.

**Comment by PrivateGER** [comment]
> 
> 
> > Probably easiest to pull the extractor from yt-dlp, now.
> 
> Hi
> I wrote the PR supporting this site [yt-dlp/yt-dlp#907](https://github.com/yt-dlp/yt-dlp/pull/907)
> While the code is released under unlicense and can be merged with yt-dl without any license issues, I'd appreciate if my code is not merged here.
> Thanks.

You probably won't have to worry anyway, the youtube-d project has been dead for months.

**Comment by dirkf** [comment]
> I wrote the PR supporting this site [yt-dlp/yt-dlp#907](https://github.com/yt-dlp/yt-dlp/pull/907)
> While the code is released under unlicense and can be merged with yt-dl without any license issues, I'd appreciate if my code is not merged here.

Having granted the Unlicense, should your code be fortunate enough that someone wants to reuse or adapt it, that person can go ahead and do so, with or without attribution. The person might refrain **if you can adduce some overriding justification**, beyond just preference, such as that the Mafia will kill your mother if the code is merged.

This is the trade-off that all contributors to the various yt-dl repositories make, in return for using code from other contributors, even more than with open source projects that have more restrictive licenses. After all, yt-dlp (2021-01...) continues yt-dlc (2020-08...2020-12) which in turn was forked from yt-dl.

**Comment by rautamiekka** [comment]
> > I wrote the PR supporting this site [yt-dlp/yt-dlp#907](https://github.com/yt-dlp/yt-dlp/pull/907)
> > While the code is released under unlicense and can be merged with yt-dl without any license issues, I'd appreciate if my code is not merged here.
> 
> Having granted the Unlicense, should your code be fortunate enough that someone wants to reuse or adapt it, that person can go ahead and do so, with or without attribution. The person might refrain **if you can adduce some overriding justification**, beyond just preference, such as that the Mafia will kill your mother if the code is merged.
> 
> This is the trade-off that all contributors to the various yt-dl repositories make, in return for using code from other contributors, even more than with open source projects that have more restrictive licenses. After all, yt-dlp (2021-01...) continues yt-dlc (2020-08...2020-12) which in turn was forked from yt-dl.

+1.

In other words: invalid argument.
